--- libwebp-0.6.1/extras/Makefile.am.omv~	2018-02-23 23:56:15.690634623 +0100
+++ libwebp-0.6.1/extras/Makefile.am	2018-02-23 23:57:34.175796031 +0100
@@ -1,12 +1,14 @@
 AM_CPPFLAGS += -I$(top_builddir) -I$(top_srcdir)
 AM_CPPFLAGS += -I$(top_builddir)/src -I$(top_srcdir)/src
-noinst_LTLIBRARIES = libwebpextras.la
+lib_LTLIBRARIES = libwebpextras.la
+
+pkginclude_HEADERS = extras.h
 
 noinst_HEADERS =
 noinst_HEADERS += ../src/webp/types.h
 
 libwebpextras_la_SOURCES =
-libwebpextras_la_SOURCES += extras.c extras.h quality_estimate.c
+libwebpextras_la_SOURCES += extras.c quality_estimate.c
 
 libwebpextras_la_CPPFLAGS = $(AM_CPPFLAGS)
 libwebpextras_la_LDFLAGS = -lm
